public/js/auth.js

```diff
@@ -55,9 +56,41 @@ if (formSignup) {
         }
         displayAlert(res.message, 2);
       })
-        .catch((err) => { displayAlert(err.message, 1); });
+      .catch((err) => { displayAlert(err.message, 1); });
     } else {
       displayAlert('Please fill in the form', 2);
     }
   });
 }
+
+if (formLogin) {
+  formLogin.addEventListener('submit', (e) => {
```

Comment:
Go and research on removeEventListener after adding

Reply:
Ok
Thanks